adapter/kubernetes/config/config.proto

```diff
@@ -92,6 +92,14 @@ message Params {
     // Default: app
     string pod_label_for_service = 6;
 
+    // In order to extract the service associated with a source, target, or
+    // origin, this adapter relies on pod labels. In particular, it looks for
+    // the value of a specific label for istio services, as specified by this
+    // parameter.
```

Comment:
nit: istio_service is a bit ambiguous, perhaps istio_component_service. 
Please add examples of "istio=ingress, istio=mixer, istio=pilot" etc.